backend/exchanges/btcdirect.go

```diff
@@ -80,14 +80,22 @@ func BtcDirectDeals() *ExchangeDealsList {
 		ExchangeName: BTCDirectName,
 		Deals: []*ExchangeDeal{
 			{
-				Fee:     3,
+				Fee:     3, // 3%
 				Payment: CardPayment,
-				IsFast:  true,
+				IsLocal: true,
```

Comment:
I think that we should keep the `isFast` too

Reply:
If we do that with the current system it would show up, but I think we don't want to have multiple isfast badges.

![Screenshot 2025-02-12 at 13 30 01](https://github.com/user-attachments/assets/56f77d61-d011-4007-ab1f-37d82959828a)